truffle/src/main/ruby/core.rb

```diff
@@ -6,6 +6,14 @@
 # GNU General Public License version 2
 # GNU Lesser General Public License version 2.1
 
+module Truffle
+  # Utility method for commenting out part of Rubinius's implementation and detailing why.  It helps clearly
+  # demarcate things we want to omit from things Rubinius has left commented out and as such, should help compare
+  # diffs when upgrading Rubinius core files.
+  def self.omit(reason)
+  end
+end
```

Comment:
Re: "more Ruby-like" ... we could just have our own top-level methods that look more like keywords.  I figured this approach looked more like code written in Ruby and handling it in the translator was an optimization pass.